packages/babel-preset-env/package.json

```diff
@@ -79,6 +78,9 @@
     "@babel/plugin-transform-unicode-regex": "workspace:^7.12.1",
     "@babel/preset-modules": "^0.1.3",
     "@babel/types": "workspace:^7.12.11",
+    "babel-plugin-polyfill-corejs2": "^0.0.8",
+    "babel-plugin-polyfill-corejs3": "^0.0.10",
+    "babel-plugin-polyfill-regenerator": "^0.0.8",
```

Comment:
Due to [how @babel/helper-define-polyfill-provider is implemented](https://github.com/babel/babel-polyfills/blob/5d2cea5ed86b4ccef4c54b03fd9a3c2791a578d2/packages/babel-helper-define-polyfill-provider/src/dependencies.js#L35) these will use `resolve` to locate modules, could they be updated to use `require.resolve` by applying the plugin you made in https://github.com/babel/babel/pull/12439 to them?